package.json

```diff
@@ -82,7 +82,7 @@
     "source-map-support": "^0.5.12",
     "ts-node": "^8.1.0",
     "tsd": "^0.22.0",
-    "typescript": "^4.1.0"
+    "typescript": "4.7.4"
```

Comment:
Pinning the typescript version - this matches what exists on `next-gen` branch  as well.